src/popup/pages/AccountDetailsNames.vue

```diff
@@ -42,12 +40,16 @@ export default {
 @use '../../styles/typography';
 
 .account-details-names {
+  --header-height: 126px;
+
   .tabs {
+    position: sticky;
+    top: calc(var(--header-height) + env(safe-area-inset-top));
     gap: var(--gap);
-    border-radius: 14px;
     padding-top: 2px;
     background-color: var(--screen-bg-color);
-    margin-bottom: 10px;
+    padding-bottom: 10px;
+    z-index: 2;
```

Comment:
I suggest to order the properties according to this rules: https://9elements.com/css-rule-order/